bee-message/src/parents.rs

```diff
@@ -68,6 +72,6 @@ impl Packable for Parents {
             inner.push(MessageId::unpack(reader)?);
         }
 
-        Ok(Self(inner))
+        Ok(Self::new(inner)?)
```

Comment:
```suggestion
        Self::new(inner)
```

Reply:
Good catch – thanks, should be addressed now.